sqladmin/templates/list.html

```diff
@@ -121,6 +124,26 @@ <h3 class="card-title">{{ model_view.name_plural }}</h3>
             <td>{{ formatted_value }}</td>
             {% endif %}
             {% endfor %}
+           {% if model_view.custom_actions_in_list %}
+               <td>
+                {% for custom_action,label in model_view.custom_actions_in_list.items() %}
+                    {% if custom_action in model_view.custom_actions_confirmation %}
+                        <a href="#" id="link-action-confirm-{{ custom_action }}" data-name="{{ custom_action }}"
+                        data-url="{{ model_view._url_for_action(row, custom_action) }}" data-bs-toggle="modal"
+                        data-bs-target="#modal-confirmation-{{ custom_action }}" class="btn btn-primary">
+                        {{ label }}
+                        </a>
+                        {% with confirmation_message = model_view.custom_actions_confirmation[custom_action], custom_action=custom_action %}
+                        {% include 'modals/confirmation.html' %}
+                        {% endwith %}
+                    {% else %}
+                        <a href="{{ model_view._url_for_action(row, custom_action) }}" class="btn btn-primary">
```

Comment:
Is this for handling the logic if no confirmation is set, then no modal is shown?

Reply:
yeah only a link is showed, and user is free to redirect to a page/show what they want. But maybe not required, or add a param in `@action` ... like you feel is better :)